components/FallbackImage.js

```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+import Image from "next/image";
+
+export const FallbackImage = ({ src, alt, fallback = "A A", ...rest }) => {
```

Comment:
I think a `default` export will be better here, but it is not a blocker to get this merged in